manageiq/providers/vmware/collector.rb

```diff
@@ -36,8 +37,9 @@ def kafka_inventory_producer
         end
 
         def publish_inventory(stream, inventory)
-          stream.produce(inventory, topic: "inventory")
-          stream.deliver_messages
+          ManageIQ::Providers::Vmware::MiqQueue.put_job(:message => inventory, :service => 'ems_operation')
```

Comment:
The service name can be anything, better to be `ems_inventory`. The subscribe needs to use the same service name.